install

```diff
@@ -296,6 +296,11 @@ Install the Linuxbrew dependencies:
 
 See http://brew.sh/linuxbrew/#dependencies for more information.
 
+Add to your #{Tty.white}.bashrc#{Tty.reset} or #{Tty.white}.zshrc#{Tty.reset}:
+  export PATH="$HOME/.linuxbrew/bin:$PATH"
+  export MANPATH="$HOME/.linuxbrew/share/man:$MANPATH"
+  export INFOPATH="$HOME/.linuxbrew/share/info:$INFOPATH"
+
```

Comment:
Perhaps give an example command that would accomplish this without opening an editor e.g. (untested)
```
by running
  echo "export PATH="$HOME/.linuxbrew/bin:$PATH" >>#{shell_profile}
```